_posts/2018-06-26-Craft-Conf.md

```diff
@@ -0,0 +1,624 @@
+
+We use estimates to help us make decisions and spark a conversation (to ultimately make a decision).
+
+#### Afraid of change
+Why is it that we want control and certainty over time, cost and schedule?
+We don't need help to make decisions, we need help to make *good* decisions.
+Are estimates then really the only way to make them?
+Are we even able to draw the right conclusions?
+Perhaps 'on time' and 'on budget' are actually not a good measure of the results of our decision if we had to cut in our feature to be able to deliver?
+
+But why hold on to estimates?
+> "Fear of losing control is a big barrier for change." - Bjarte Bogsnes
+
+We tend to hold on to what we know.
+A few quotes in favor of breaking the cycle:
+
+> "Alternative to estimates: do the most important thing until either it ships or it is no longer the most important thing." - Kent Beck
+
+> "As teams progress they first struggle with estimation, then can get quite good at it, and then reach a point where they often don't need it." - Martin Fowler
+
+When starting a project, requirements are handed over to the development teams in an orderly fashion, very neatly organized and containing everything that needs to happen.
+Or so the stakeholders like to think.
+The project itself, however, usually ends up being a lot more disorganized with features being added, removed or changed with no way to tell early on what the end result will look like.
+Then why try to get estimates for the project if most of the requirements tend to change anyway?
+
+> "It's in the doing of the work that we discover the work that we must do." - Woody Zuill
+
+Rather than size and smallness, look for the following qualities in a story:
+* Potentially valuable
+* Understandable
+* Cohesive - does it belong together
+* Distinct - is it clearly separate from other stories
+-> These don't require estimates! When these qualities are found, we have something we can work on.
+
+#### The Twelve Calculations example
+* 80% of the use of the app comes from 20% of the features
+* 80% of the use of a feature comes from 20% of its implementation
+
+This means that with only 4% of the implementation effort (20% of 20%), we can cover the 20% of the features being actively used.
+
+Let's try to have many, small, inexpensive attempts at value.
+Let's do, discover, validate and steer.
+This is what being agile is all about!
+
+What do you think would have more payoff?
+Turning up the goods on getting our estimates better?
+Or turning up the goods on our ability to rapidly deliver potentially useful software?
+
+Let's learn to control our urge to control things.
+Let's quit worrying about whether we will get done in 3 months for now.
+Let's get good at being done everyday.
+
+On [Woody's website](http://zuill.us/WoodyZuill/beyond-estimates/){:target="_blank"}, you can find more info on his take on estimates.
+
+Thanks for reading, everyone!
```

Comment:
I'd put this in a different section because right now it appears like it's part of the last talk?